content/en/load-testing/11-glossary.md

```diff
@@ -718,8 +717,9 @@ certain computer.
 ## Test Case (XLT)
 
 Using XLT, a test case is a piece of code that implements a user
-transaction and that can be run with the XLT script developer (xml test
-scripts) or with any JUnit test runner (Java test case classes).
+transaction and that can be run with any JUnit test runner (Java test case classes) 
```

Comment:
can be executed as a JUnit test
Why: We bring our own runner hence you cannot use any other runner.

Reply:
The testcase itself is just a unit test that can be run from IDE or console as well.